settings.gradle

```diff
@@ -192,3 +194,186 @@ if (JavaVersion.current() == JavaVersion.VERSION_1_8) {
   }
 }
 
+dependencyResolutionManagement {
+  versionCatalogs {
+    libs {
+      String antlrVersion = "4.8"
+      String awsSdkVersion = "2.20.18"
+      String flink15Version = "1.15.0"
+      String flink16Version = "1.16.1"
+      String flink17Version = "1.17.0"
+      String hadoop2Version = "2.7.3"
+      String hive2Version = "2.3.9"
+      String hive3Version = "3.1.3"
+      String jackson11Version = "2.11.4"
```

Comment:
nit: maybe `jackson211Version` instead of `jackson11Version`? Same for the others

Reply:
yeah, I was hesitant about that because of potential Jackson major version change but Jackson version 2 was released more than 10 years ago so I decided it doesn't change that frequently
we can change those to jackson2xxVersion if you think it worth it